demo/repo/flow.groovy

```diff
@@ -3,54 +3,61 @@ node {
 
   def maven = docker.image('maven:3.3.9-jdk-8'); // https://registry.hub.docker.com/_/maven/
 
-  stage 'Mirror'
-  // First make sure the slave has this image.
-  // (If you could set your registry below to mirror Docker Hub,
-  // this would be unnecessary as maven.inside would pull the image.)
-  maven.pull()
+  stage('Mirror') {
+    // First make sure the slave has this image.
+    // (If you could set your registry below to mirror Docker Hub,
+    // this would be unnecessary as maven.inside would pull the image.)
+    maven.pull()
```

Comment:
[for reviewers](https://github.com/jenkinsci/docker-workflow-plugin/pull/62/files?w=1)